parmed/openmm/parameters.py

```diff
@@ -214,6 +214,13 @@ def _find_unused_types(self, skip_residues):
                     keep_types.add(atom.type)
         return {typ for typ in self.atom_types if typ not in keep_types}
 
+    def _templhasher(self, residue):
```

Comment:
This never uses `self` -- either make it a `@staticmethod` or a module-level global function.